### PR TITLE
Add a backend option to exclude publishers for an advertiser

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: adserver/models.py [hunk 1]
```diff
@@ -305,6 +305,12 @@ class Campaign(TimeStampedModel, IndestructibleModel):
         ),
     )
 
+    exclude_publishers = models.ManyToManyField(
```
Comment (ericholscher) [inline adserver/models.py:308]: Is there a reason we have this on the Campaign and not at the Flight level? I guess we're doing the Campaign <-> Publisher mapping here as well with publisher groups, so it makes sense, but I definitely found it hard to follow the mapping of this stuff having it in a different place than all the other targeting information on the Flight.

**Review by ericholscher — APPROVED** [review]
Seems simple enough.

File: adserver/models.py (hunk 1 above)
Comment (davidfischer) [reply adserver/models.py:308]: It doesn't matter too much but the way I think of it is that things that should almost never change go on the Campaign while things that might change from one ad buy to the next go on the flight. I'm open to moving it.

Comment (ericholscher) [reply adserver/models.py:308]: Not a huge deal, thanks for explaining the mental model, it makes sense. 👍
